src/python/include/Persistent_cohomology_interface.h

```diff
@@ -73,6 +73,60 @@ persistent_cohomology::Persistent_cohomology<FilteredComplex, persistent_cohomol
     return persistence;
   }
 
+  void top_dimensional_cofaces(std::vector<int> & cofaces, int splx){
+    if (stptr_->dimension(stptr_->simplex(splx)) == stptr_->dimension()){cofaces.push_back(stptr_->simplex(splx));}
+    else{  for (auto v : stptr_->coboundary_simplex_range(stptr_->simplex(splx))){top_dimensional_cofaces(cofaces, stptr_->key(v));}  }
+  }
+
+  std::vector<std::pair<int, std::pair<std::pair<double, int>, std::pair<double, int>>>> get_cofaces_of_cubical_persistence_pairs(int homology_coeff_field,
+                                                                                                                                  double min_persistence) {
+
+    // Warning: this function is meant to be used with CubicalComplex only!!
+
+    // Gather all top-dimensional cells and store their simplex handles
```

Comment:
There is `top_dimensional_cells_range()` if that helps.

Reply:
Cool, thanks!